Dockerfile

```diff
@@ -0,0 +1,39 @@
+ARG JAVA_VERSION=16
+ARG JVM_FLAVOR=hotspot
+
+FROM adoptopenjdk:${JAVA_VERSION}-jdk-${JVM_FLAVOR} AS builder
+WORKDIR /builddir
+
+COPY gradlew build.gradle settings.gradle ./
+COPY gradle/ gradle/
+RUN ./gradlew --no-daemon # Download gradle and init
+
+COPY license*.txt ./
+COPY src/ src/
+RUN ./gradlew clean buildForDocker --no-daemon
+
+
+ARG JAVA_VERSION
+ARG JVM_FLAVOR
+
+FROM adoptopenjdk:${JAVA_VERSION}-jre-${JVM_FLAVOR}
+WORKDIR /app
+
+RUN groupadd --system spring \
```

Comment:
Commonly one would prefer the app name as group & user (i.e. `bibliothek:bibliothek`), but not a blocker in any sense.